README.md

```diff
@@ -122,6 +92,34 @@ node.auth.hook(function (auth, args) {
 })
 ```
 
+### `app.close()`
+
+close the app!
+
+Optionally takes `(err, callback)`
+
+----
+
+## TODO document
```

Comment:
it's good for a reader to know that there are things that are not documented yet, but in the interest of actually getting these documented, it's better if they are in an issue